src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -56,13 +56,15 @@ public MetaTileEntity createMetaTileEntity(MetaTileEntityHolder holder) {
     public NBTTagCompound writeToNBT(NBTTagCompound data) {
         super.writeToNBT(data);
         data.setTag("ContainerInventory", containerInventory.serializeNBT());
+        //fluidTank.writeToNBT(data);
```

Comment:
Is this code supposed to have a purpose?

Reply:
I just suspect it might be needed, but it seems to work without it so I left the commented code there in case an issue comes up in the future